src/theme/components/hero.ts

```diff
@@ -105,8 +105,65 @@ const getSecondaryVariantStyles = (bgColor: string = "") => {
     },
   };
 };
+const getTextColor = (type, mode, foregroundColor, isDarkText) => {
+  const prefix = mode === "dark" ? "dark.ui." : "ui.";
+  const colorLight = foregroundColor
+    ? foregroundColor
+    : isDarkText
+    ? `${prefix}typography.${type}`
+    : `${prefix}typography.inverse.${type}`;
+  const colorDark = foregroundColor
+    ? foregroundColor
+    : isDarkText
+    ? `${prefix}typography.inverse.${type}`
+    : `${prefix}typography.${type}`;
+  const colorLink = foregroundColor
+    ? foregroundColor
+    : isDarkText
+    ? `ui.link.primary`
+    : `dark.ui.link.primary`;
+  const finalColor =
+    type === "link" ? colorLink : mode === "dark" ? colorDark : colorLight;
```

Comment:
You don't have an example where "link" is being passed to the `getTextColor` function.

Can you write a few tests for these utility functions? Do you think it's worth it to verify the values returned are the expected ones?

Reply:
Seeing that I also have a `getLinkColor` function, I must have initially added this condition here and then realized I needed something more powerful.

I will see what I can do with the tests.